draft-wang-ppm-dap-taskprov.md

```diff
@@ -296,6 +296,27 @@ A protocol participant MUST opt out if the task has expired.
 The behavior of each protocol participant is determined by whether or not they
 opt in to a task.
 
+## Supporting a Global HPKE Configuration {#global-hpke-config}
+
+In DAP, Clients need to know the HPKE configuration of each Aggregator before
+sending reports. (See HPKE Configuration Request in
+{{!DAP=I-D.draft-ietf-ppm-dap-02}}.) However, in a DAP deployment that supports
+`task_prov` extension, if a Client queries an Aggregator's `/hpke_config`
+endpoint with a task ID computed as described in {{construct-task-id}}, the
+task ID may not be configured in the Aggregator yet, because the Aggregator is
+still waiting for the first Client report with the `task_prov` extension
+to arrive.
+
+To mitigate this issue, if an Aggregator wants to support `task_prov` extension,
+it SHOULD support a global HPKE configuration independent of the task ID, for all
```

Comment:
Re: https://github.com/wangshan/draft-wang-ppm-dap-taskprov/pull/21#issuecomment-1271070699

The problem is the guidance "SHOULD support a global HPKE configuration". We don't want to limit the Aggregator to just one configuration, for the following reasons:
1. It may want to support multiple HPKE cipher suites (say, X25519 and P256)
2. It may want to support multiple HPKE keys at the same time (say, to account for clients that may have used an older config)

Something like this might be better: "SHOULD choose which HPKE configuration to advertise to clients independent of the task ID."

Reply:
I see, I didn't realize that is not clear. Maybe we should avoid the phrase "a global HPKE configuration".

I applied your suggestion, and also updated the title of this section to `Supporting HPKE Configurations Independent of Tasks {#hpke-config-no-task-id}`. Let me know if that doesn't make sense.